<launch>
	<node name="autonomy_node" pkg="autonomy" type="autonomy_node.py" launch-prefix="xterm -e"/>
	<node name="visualizer_node" pkg="autonomy" type="visualizer.py"/>
</launch>
